bootstrap/eks/README.md

```diff
@@ -0,0 +1,54 @@
+# Cluster API bootstrap provider EKS
+
+Cluster API bootstrap provider EKS (CABPE) is a component of [Cluster API](https://github.com/kubernetes-sigs/cluster-api/blob/master/README.md) that is responsible for generating a cloud-init script to turn a Machine into a Kubernetes Node; this implementation uses the [AWS-provided EKS bootstrap script](https://github.com/awslabs/amazon-eks-ami/blob/master/files/bootstrap.sh) for joining Kubernetes Nodes to EKS clusters. 
+
+CABPE is the bootstrap component of Cluster API Provider AWS' (CAPA) EKS ecosystem. This ecosystem is comprised of:
+- EKS controlplane provider (AWSManagedControlPlane)
+- AWS infrastructure provider (AWSMachine, AWSMachineTemplate, AWSMachinePool)
+- EKS bootstrap provider (EKSConfig, EKSConfigTemplate)
+
+## How does CABPE work?
+
+CABPE generates cloud-init configuration that CAPA will use as the [user-data](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html) for new EC2 instances. The generated configuration relies on the `bootstrap.sh` script that is present on all official AWS EKS AMIs. 
+
+The output script looks something like this (assuming an EKS cluster with the name `my-cluster`):
+
+```bash
+#!/bin/bash
+/etc/eks/bootstrap.sh my-cluster
+```
+
+CAPA will look up official the official EKS AMI for a particular Kubernetes version if the AMI is not overriden in the AWSMachine object. In order for this bootstrap provider to function, any user-specified AMIs will need to have `/etc/eks/bootstrap.sh` present (script source linked above).
```

Comment:
Small nit `look up official the official EKS AMI` to `look up the official EKS AMI`